applications/pallet_supersig.md

```diff
@@ -0,0 +1,306 @@
+			supersig_id: T::AccountId,
+			call_index: CallIndex,
+		)
+```
+
+**Add member to the supersig**
+
+```rust
+pub fn add_member(
+      origin: OriginFor<T>,
+			supersig_id: T::AccountId,
+			call_index: CallIndex,
+		)
+```
+
+**Remove member from the supersig**
+```rust
+pub fn remove_member(
+      origin: OriginFor<T>,
+			supersig_id: T::AccountId,
+			call_index: CallIndex,
+		)
+```
+
+**Milestone 1** to make a working, secure and usable pallet_supersig v0.1.0. 
+
+- The pallet will be tested on the Pop-Art, community run relay test network
+- The pallet will be well benchmarked. 
+- The pallet will be functioning in both Kabocha and Edgware chains at the minimum.
+- Unit integration tests
+- Easy to spin up node to test pallet 
+- Clearly written as Readme and inline documentation
+
+**Milestone 2** will make the Supersig as visible of an experience within the polkadot js UI as Multisig, The PR will be made, and if there is a blocker, then the apps.decentration.org fork will house the feature. 
+
+- Polkadot JS UI will be forked and adapted to show the Supersig IF the chain has the pallet installed.
+
+**Milestone 3** Will get feedback and make some improvements. One improvement is adding a "Super Beneficiary" option that allows a threshold of 2/n only if the Super Beneficiary is involved in the transaction. Else the transactio will default to a superMajority. This allows for greater than the usual 2/3 multisig, but without loss of security. 
+
+
+> :zap: If any of your deliverables is based on somebody else's work, make sure you work and publish _under the terms of the license_ of the respective project and that you **highlight this fact in your milestone documentation** and in the source code if applicable! **Teams that submit others' work without attributing it will be immediately terminated.**
+### Overview
+
+- **Total Estimated Duration:** 5 to 7 weeks
+- **Full-Time Equivalent (FTE):**  2
+- **Total Costs:** USD 27,000
+
+### Milestone 1 — pallet_supersig MVP
+
+- **Estimated duration:** 3 to 4 weeks
+- **FTE:**  2
+- **Costs:** 14,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | Apache 2.0  |
+| 0b. | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| 0c. | Testing Guide | Core functions will be fully covered by unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an **article**/workshop that explains [...] (what was done/achieved as part of the grant). (Content, language and medium should reflect your target audience described above.)
```

Comment:
Can you please remove the template text from all `Article` deliverables and replace them with what you are actually planning to publish?